C/services/south/ingest.cpp

```diff
@@ -452,6 +455,12 @@ void Ingest::waitForQueue()
  */
 void Ingest::processQueue()
 {
+
+	typedef struct{
+                std::set<std::string> setDp;
+                unsigned int          count;
```

Comment:
count needs to be maintained for each dplist we are maintaining.

Reply:
fixed